petclinic.yml

```diff
@@ -9,12 +10,18 @@ application:
       application-pic: bind(haproxy#app.petclinic-image)
   configuration:
     configuration.clusterSize: 1
+    configuration.haproxy-docker-image-id: "qubell/haproxy:latest"
+    configuration.database-initializer-docker-image-id: "tutum/mysql"
```

Comment:
No, we don't. We can keep it as a configuration pin, but do not map this pin to app root.